src/dbus/abrt-configuration.c

```diff
@@ -1,1056 +0,0 @@
-#include <gio/gio.h>
```

Comment:
> the service was never used anywhere since its inception five years ago 

Do you mean never used anywhere in the code or by the users/customers? 

I see 4 issues related to this, so this was probably requested by someone. 
- https://github.com/abrt/abrt/issues/750
- https://github.com/abrt/abrt/issues/751
- https://github.com/abrt/abrt/issues/752
- https://github.com/abrt/abrt/issues/754

> it only makes maintenance more difficult

Just because something is difficult to maintain shouldn't be the only reason to remove it.

Reply:
If no one is using it, then it’s perfectly fine to drop some load… But that is a different question.

Reply:
It's not used anywhere by us, as far as I know. It doesn't seem to be used in any public package. I have just sent an inquiry into the Fedora devel mailing list to see if anybody is using or relying on it privately.

Reply:
I received a private message from @jfilak, the original author, asserting that the daemon was never used for its intended purpose anyway. Let's wait some more if anybody else responds on the mailing list, although I find it unlikely.